src/runtime/types.cpp

```diff
@@ -4399,6 +4421,12 @@ void setupRuntime() {
     type_cls->giveAttrDescriptor("__abstractmethods__", (getter)type_abstractmethods, (setter)type_set_abstractmethods);
 
     type_cls->giveAttr("__call__", new BoxedFunction(typeCallObj));
+    type_cls->giveAttr(
+        "__subclasscheck__",
+        new BoxedFunction(BoxedCode::create((void*)type___subclasscheck__, BOXED_BOOL, 2, "type.__subclasscheck__")));
```

Comment:
I think this should be, `BoxedCode::create((void*)type___subclasscheck__, BOXED_BOOL, 2, "type.__subclasscheck__", ParamNames::empty(), CAPI)`